src/adapter/events.ts

```diff
@@ -34,18 +33,10 @@ type DeviceLeavePayload = {
 };
 
 interface ZclDataPayload {
-    address: number | string;
-    frame: ZclFrame;
-    endpoint: number;
-    linkquality: number;
-    groupID: number;
-    wasBroadcast: boolean;
-    destinationEndpoint: number;
-}
-
-interface RawDataPayload {
     clusterID: number;
     address: number | string;
+    zclFrameHeader: ZclHeader | undefined;
```

Comment:
Can we just name this `header`? Makes for shorter variables everywhere, and quick one-word assignment to objects.
I'd even say we can rename `ZclDataPayload` to just `ZclPayload` while we're at it.

```typescript
const header: ZclHeader;

try {
    header = ...;
} ...

const payload: ZclPayload = {
    header,
    ...
};
```

Reply:
Definitely, will do it once all tests are fixed.

Reply:
Done!